cfgov/unprocessed/js/modules/util/email-popup-helpers.js

```diff
@@ -1,8 +1,18 @@
 import { assign } from './assign';
 import throttle from 'lodash.throttle';
+import webStorageProxy from './web-storage-proxy';
+
+// If cookies are turned off, we set localStorage variables to an empty object.
+let _localStorage;
+
+try {
+  _localStorage = window.localStorage;
+} catch ( err ) {
+  _localStorage = {};
```

Comment:
Can this be set to `null` for parity with how it's handled in analytics-gtm?

Reply:
Unlike analytics-gtm, which bails out if `_localStorage` is `null`, here it's passed into `webStorageProxy` in multiple methods. So if we wanted email helpers and analytics-gtm to both be null or both be empty objects, we either would need logic to check if it's `null` in each method in the email popup helper, which doesn't seem great, or we make it an object in analytics-gtm and check if it's an empty object, which doesn't seem great either. So that's why it's different in each.

Reply:
Then begrudgingly approved!

Reply:
@wpears ! I thought about this further and added a null check inside the web-storage-proxy, now it can be null in email popup helpers! https://github.com/cfpb/consumerfinance.gov/pull/5953/commits/4d3627c2a53aae9dd5a5db2a7312edeae948d0b0